frontend/src/assets/cohorts.json

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
Can't this be retrieved from the backend?

Reply:
The information stored here is coming from the publications/cohort websites, not the actual data we got.